src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -198,7 +213,7 @@ public void setAuthorList(List<Author> authorList) {
     public void resetAuthorInformation() {
         authorEmailsAndAliasesMap.clear();
         authorDisplayNameMap.clear();
-        authorList.forEach(this::setAuthorDetails);
+        authorList.forEach(author -> setAuthorDetails(author));
```

Comment:
is this change needed?

Reply:
I did the change thinking code will be more readable. But can remove too.

Reply:
dont think its necessary for this change